sky/utils/resources_utils.py

```diff
@@ -0,0 +1,63 @@
+"""Resources related utilities."""
+from sky.clouds import service_catalog
+from sky.utils import ux_utils
+
+# List of all accelerators supported by SkyPilot.
+# NOTE: Must include accelerators supported for local clusters.
+_ACCELERATORS = [
```

Comment:
Comment that these are the *canonicalized* accelerator names used internally. Users should call `canonicalize_accelerator_name()` to get them.

Reply:
Fixed.